portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -346,4 +381,25 @@ definition {
 			value1 = "${taskCount}");
 	}
 
+	macro waitForSLAtoRun {
+		var count = "0";
+
+		while (IsElementPresent(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_OVERDUE_ITEMS_VALUE_NONE") && ("${count}" != "12")) {
+
+			echo("Waiting 5 seconds, until the SLA becomes overdue");
+
+			Pause(locator1 = "5000");
+
+			// //Add a function here to refresh the page. We need to search for one.
+
+			WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+			count = MathUtil.sum("${count}", "1");
+		}
+
+		if ("${count}" == "15") {
+			echo("timeout exceeded");
+		}
+	}
+
```

Comment:
hey man, an excellent job you did here.

While reviewing your code, I noticed that we can improve 2 things:
1) change the name of the macro to explicit that we are waiting for the task to become overdue.

2) Add a parameter for the time we want to wait. The seconds we want to wait could be passed on the .testcase and then incremented from 0 until the max time. Like that, we will make it flexible to use any amount of seconds we want, instead of delimiter 60 seconds

3) Delete the line #393 with the comment

But, if you think that 2) is requires too much effort for you, you can let the way it is and we can think on a solution later

Thank you very much for doing it, man.